doc/classes/@GlobalScope.xml

```diff
@@ -1414,6 +1414,44 @@
 		<constant name="KEY_F16" value="16777259" enum="Key">
 			F16 key.
 		</constant>
+		<constant name="KEY_F17" value="16777260" enum="Key">
+		</constant>
+		<constant name="KEY_F18" value="16777261" enum="Key">
+		</constant>
+		<constant name="KEY_F19" value="16777262" enum="Key">
+		</constant>
+		<constant name="KEY_F20" value="16777263" enum="Key">
+		</constant>
+		<constant name="KEY_F21" value="16777264" enum="Key">
+		</constant>
+		<constant name="KEY_F22" value="16777265" enum="Key">
+		</constant>
+		<constant name="KEY_F23" value="16777266" enum="Key">
+		</constant>
+		<constant name="KEY_F24" value="16777267" enum="Key">
+		</constant>
+		<constant name="KEY_F25" value="16777268" enum="Key">
+		</constant>
+		<constant name="KEY_F26" value="16777269" enum="Key">
+		</constant>
+		<constant name="KEY_F27" value="16777270" enum="Key">
+		</constant>
+		<constant name="KEY_F28" value="16777271" enum="Key">
+		</constant>
+		<constant name="KEY_F29" value="16777272" enum="Key">
+		</constant>
+		<constant name="KEY_F30" value="16777273" enum="Key">
+		</constant>
+		<constant name="KEY_F31" value="16777274" enum="Key">
+		</constant>
+		<constant name="KEY_F32" value="16777275" enum="Key">
+		</constant>
+		<constant name="KEY_F33" value="16777276" enum="Key">
+		</constant>
+		<constant name="KEY_F34" value="16777277" enum="Key">
+		</constant>
+		<constant name="KEY_F35" value="16777278" enum="Key">
+		</constant>
```

Comment:
```suggestion
		<constant name="KEY_F17" value="16777260" enum="Key">
			F17 key.
		</constant>
		<constant name="KEY_F18" value="16777261" enum="Key">
			F18 key.
		</constant>
		<constant name="KEY_F19" value="16777262" enum="Key">
			F19 key.
		</constant>
		<constant name="KEY_F20" value="16777263" enum="Key">
			F20 key.
		</constant>
		<constant name="KEY_F21" value="16777264" enum="Key">
			F21 key.
		</constant>
		<constant name="KEY_F22" value="16777265" enum="Key">
			F22 key.
		</constant>
		<constant name="KEY_F23" value="16777266" enum="Key">
			F23 key.
		</constant>
		<constant name="KEY_F24" value="16777267" enum="Key">
			F24 key.
		</constant>
		<constant name="KEY_F25" value="16777268" enum="Key">
			F25 key. Only supported on macOS and Linux.
		</constant>
		<constant name="KEY_F26" value="16777269" enum="Key">
			F26 key. Only supported on macOS and Linux.
		</constant>
		<constant name="KEY_F27" value="16777270" enum="Key">
			F27 key. Only supported on macOS and Linux.
		</constant>
		<constant name="KEY_F28" value="16777271" enum="Key">
			F28 key. Only supported on macOS and Linux.
		</constant>
		<constant name="KEY_F29" value="16777272" enum="Key">
			F29 key. Only supported on macOS and Linux.
		</constant>
		<constant name="KEY_F30" value="16777273" enum="Key">
			F30 key. Only supported on macOS and Linux.
		</constant>
		<constant name="KEY_F31" value="16777274" enum="Key">
			F31 key. Only supported on macOS and Linux.
		</constant>
		<constant name="KEY_F32" value="16777275" enum="Key">
			F32 key. Only supported on macOS and Linux.
		</constant>
		<constant name="KEY_F33" value="16777276" enum="Key">
			F33 key. Only supported on macOS and Linux.
		</constant>
		<constant name="KEY_F34" value="16777277" enum="Key">
			F34 key. Only supported on macOS and Linux.
		</constant>
		<constant name="KEY_F35" value="16777278" enum="Key">
			F35 key. Only supported on macOS and Linux.
		</constant>
```

Reply:
It was easier for me to do this from my computer, but I did it :)